Console/Command/Task/IndexTask.php

```diff
@@ -35,7 +35,11 @@ public function execute() {
                     App::uses($model, 'Model');
                     if (class_exists($model)) {
                         $tmpModel = new $model();
-                        if (isset($tmpModel->actsAs) && is_array($tmpModel->actsAs) && isset($tmpModel->actsAs['Searchable.Searchable'])) {
+                        # this is flawed and will only work if you set options (i.e. fields) for the Searchable behavior
+                        # if (isset($tmpModel->actsAs) && is_array($tmpModel->actsAs) && isset($tmpModel->actsAs['Searchable.Searchable'])) {
+                        # below is a more accurate way to determine whether or not a particular behavior exists
+                        $behaviors = $tmpModel->Behaviors->loaded();
```

Comment:
In my opinion - this shouldn't stay in the code... you can leave a comment about what the code is doing, but don't leave commented out code with an explanation about why it doesn't work... that just clutters things up...  

That said, I agree with this change